scripts/agent-evals/src/mock/mock-tools-main.ts

```diff
@@ -67,6 +65,10 @@ Module.prototype.require = function (id: string) {
   });
 };
 
+import { getToolMocks } from "./tool-mocks.js";
+
+const mocks = getToolMocks();
```

Comment:
Curious why this was moved? I think it still needs to be up at the top of the file if I'm not mistaken